sites/partners/src/listings/PaperListingForm/index.tsx

```diff
@@ -148,7 +169,29 @@ const ListingForm = ({ listing, editMode }: ListingFormProps) => {
       const typedData: FormListing = {
```

Comment:
Maybe we can create a separate function to create the request body in a good shape (move all typedData logic). It could reduce the amount of `onSubmit` function complexity.